api/v1alpha1/gatekeeper_types.go

```diff
@@ -86,6 +87,19 @@ type GatekeeperSpec struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Pod Annotations"
 	// +optional
 	PodAnnotations map[string]string `json:"podAnnotations,omitempty"`
+
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Config Configuration"
+	// +optional
+	Config *ConfigConfig `json:"config,omitempty"`
```

Comment:
Why is this a pointer? I don't think we need to check if it was explicitly set or not.

Reply:
To check ConfigConfig null or not

Reply:
Okay, I see. We need to know this for the upgrade case.